bom-2.204.x/pom.xml

```diff
@@ -9,7 +9,7 @@
     <artifactId>bom-2.204.x</artifactId>
     <packaging>pom</packaging>
     <properties>
-        <configuration-as-code-plugin.version>1.36</configuration-as-code-plugin.version>
+        <configuration-as-code-plugin.version>1.37-20201006.192214-1</configuration-as-code-plugin.version>
```

Comment:
No incremental version?

Reply:
Can't becase incremental only works when in line with master branch...

Reply:
As demonstrated by my first build because my master branch was behind locally: https://ci.jenkins.io/blue/organizations/jenkins/Plugins%2Fconfiguration-as-code-plugin/detail/PR-1477/1/pipeline#step-110-log-23

Reply:
> incremental only works when in line with master branch

Not true. It is only required for the PR branch head to be a descendant of the base branch head.

Reply:
I never had it working with anything directly in front of master. Even though that specific build my master branch was clean and was a descendant of master upstream branch.

Reply:
https://github.com/jenkinsci/configuration-as-code-plugin/pull/1477 is on `master` and is 1.44-SNAPSHOT. That got deployed fine. My point is that if you want a deployment from some backport branch, you need to create the backport branch, pushing it to the origin repo, then create a PR based on that branch. I do not see any such PR.